system/system_error_monitor/config/system_error_monitor.param.yaml

```diff
@@ -20,6 +20,7 @@
         /autoware/control/autonomous_driving/node_alive_monitoring: default
         /autoware/control/autonomous_driving/performance_monitoring/lane_departure: default
         /autoware/control/control_command_gate/node_alive_monitoring: default
+        /autoware/control/autonomous_emergency_braking/performance_monitoring/emergency_stop: { sf_at: "none", lf_at: "warn", spf_at: "error", auto_recovery: "false"}
```

Comment:
@purewater0901 AEB node seems to be designed to publish OK and ERROR only.
What do you mean by setting `warn` value to `lf_at`?
It is OK that the `warn` is reserved value for future functionality extension to realize a certain use case in the future.

Reply:
I talked to @purewater0901 internally and this `warn` is reserved value to extend functionality of AEB in the future.